kitchen.yml

```diff
@@ -44,7 +44,7 @@ platforms:
         - sh bootstrap-salt.sh -XdPbfrq -x python3 git develop
       run_command: /usr/lib/systemd/systemd
 
-  ## SALT 2019.2
+  ## SALT `2019.2`
```

Comment:
Is this  intended or did you mean `'2019.2'`
```
## SALT `2019.2` 
```

Reply:
@noelmcloughlin No, that was intended.  To be fair, it's a particular idiosyncrasy of mine, when highlighting code-like elements when discussing in Markdown-enabled environments (as you may have noticed when we have discussions on Slack).  Since we're leading to a central controlling formula (`ssf-formula`), we can always argue it out there and then propagate any changes easily.